mmv1/products/compute/InterconnectAttachmentGroup.yaml

```diff
@@ -0,0 +1,258 @@
+update_verb: 'PATCH'
+timeouts:
+  insert_minutes: 20
+  update_minutes: 20
+  delete_minutes: 20
+async:
+  actions: ['create', 'delete', 'update']
+  type: 'OpAsync'
+  operation:
+    base_url: '{{op_id}}'
+  result:
+    resource_inside_response: false
+examples:
+  - name: 'interconnect_attachment_group_basic'
+    primary_resource_id: 'example-interconnect-attachment-group'
+    vars:
+      interconnect_attachment_group_name: 'example-interconnect-attachment-group'
+      deletion_protection: 'true'
+    test_vars_overrides:
+      'deletion_protection': 'false'
+parameters:
+properties:
+  - name: 'description'
+    type: String
+    description: |
+      An optional description of this resource. Provide this property when you create the resource.
+  - name: 'creationTimestamp'
+    type: Time
+    description: |
+      Creation timestamp in RFC3339 text format.
+    output: true
+  - name: 'name'
+    type: String
+    description: |
+      Name of the resource. Provided by the client when the resource is created. The name must be
+      1-63 characters long, and comply with RFC1035. Specifically, the name must be 1-63 characters
+      long and match the regular expression `[a-z]([-a-z0-9]*[a-z0-9])?` which means the first
+      character must be a lowercase letter, and all following characters must be a dash,
+      lowercase letter, or digit, except the last character, which cannot be a dash.
+    required: true
+    immutable: true
+    validation:
+      regex: '^[a-z]([-a-z0-9]*[a-z0-9])?$'
+  - name: 'attachments'
+    type: Map
+    description: |
+      Attachments in the AttachmentGroup. Keys are arbitrary user-specified
+      strings. Users are encouraged, but not required, to use their preferred
+      format for resource links as keys.
+      Note that there are add-members and remove-members methods in gcloud.
+      The size of this map is limited by an "Attachments per group" quota.
+    key_name: 'name'
+    key_description: |
+      Resource name
+    value_type:
+      name: mapObjectName
+      type: NestedObject
+      properties:
+        - name: 'attachment'
+          type: String
+  - name: 'interconnectGroup'
+    type: String
+    description: |
+      The URL of an InterconnectGroup that groups these Attachments'
+      Interconnects. Customers do not need to set this unless directed by
+      Google Support.
+  - name: 'intent'
+    type: NestedObject
+    description: |
+      The user's intent for this group. This is the only required field besides
+      the name that must be specified on group creation.
+    properties:
+      - name: 'availabilitySla'
+        type: Enum
+        description: |
+          Which SLA the user intends this group to support.
+        enum_values:
+          - 'PRODUCTION_NON_CRITICAL'
+          - 'PRODUCTION_CRITICAL'
+          - 'NO_SLA'
+          - 'AVAILABILITY_SLA_UNSPECIFIED'
+    required: true
+  - name: 'logicalStructure'
```

Comment:
Looking at the docs `logicalStructure` appears to be completely output only, so you'll have to add `output: true` to all of the subfields too (it doesn't propagate from the top level field)